src/Core/src/ImageSources/FileImageSourceService/FileImageSourceService.iOS.cs

```diff
@@ -21,11 +21,12 @@ public partial class FileImageSourceService
 
 			try
 			{
-				var image = imageSource.GetPlatformImage();
-
-				if (image == null)
+				using var cgImageSource = imageSource.GetPlatformImageSource(out var loadedScale);
```

Comment:
Are there any existing issues that this addresses? It seems like we're slightly changing what image we're loading?

Reply:
Previously the `GetPlatformImage` used `UIImage.FromBundle` API which loaded the image file based on the scale (so "myimage.png" could be loaded using "myimage@3x.png").in the new API you need to provide the exact path, so we need to pre calculate the @3x and then load that directly. `FromBundle` seems to have just used the main screen scale, so we use that in the new code.

An open question is whether or not we should ignore that and use the scale provided by the service itself.